reference/course.v1.yaml

```diff
@@ -90,9 +98,11 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/StreamCourse'
+                $ref: '#/components/schemas/Course'
       operationId: get-stream-course
-      description: Get a specific course by stream ID - returns basic metadata.
+      description: |-
+        Get a specific course by stream ID - returns basic metadata.
+        Usually the course is found via the catalog, each product has a list of courses that is available for a student/teacher when that product is bought. This call is used to get more infomation, can also be used to see if there are any new versions of this course published
   '/course/{id}/cmi5e-course':
```

Comment:
wondering whether the -course has any added value, it is already in the context of course/{id}